rslib/Cargo.toml

```diff
@@ -1,9 +1,11 @@
 [package]
 name = "anki"
-version = "2.1.24" # automatically updated
+version = "2.1.24"  # automatically updated
 edition = "2018"
-authors = ["Ankitects Pty Ltd and contributors"]
+authors = ["Ankitects Pty Ltd and contributors <pypi@cargo.bug>"]
```

Comment:
Is this mail address here intended? I'd expect this to rather point to Anki support page

Reply:
I do not know any anki support email and pypi is rejecting the wheel if there is not email.

Reply:
@dae, is there an email for support?